examples/Cassie/run_dircon_squatting.cc

```diff
@@ -97,11 +97,15 @@ using dairlib::multibody::GetBodyIndexFromName;
 DEFINE_string(init_file, "", "the file name of initial guess");
 DEFINE_string(data_directory, "../dairlib_data/cassie_trajopt_data/",
               "directory to save/read data");
-DEFINE_bool(store_data, false, "To store soluation or not");
+DEFINE_bool(store_data, false, "To store solution or not");
 DEFINE_int32(max_iter, 100000, "Iteration limit");
 DEFINE_double(duration, 0.4, "Duration of the single support phase (s)");
 DEFINE_double(tol, 1e-4, "Tolerance for constraint violation and dual gap");
 
+// Parameters which enable dircon-improving features
+DEFINE_bool(is_scale_constraint, true, "Scale the nonlinear constraint values");
+DEFINE_bool(is_scale_variable, false, "Scale the decision variable");
```

Comment:
Is false the right default?

Reply:
I set it to false, because it runs slower with variable scaling in this example. It is possible that the initial guess in the example is very good, so the variable scaling doesn't help much. (it's hard to find the optimal scaling factors that improve the performance)